Introducing_CircuitPlaygroundExpress/CircuitPlaygroundExpress_CapTouch.py

```diff
@@ -1,32 +1,31 @@
-# CircuitPlaygroundExpress_CapTouch
+# Circuit Playground Capacitive Touch
 
 import time
-
 import board
 import touchio
 
-touch1 = touchio.TouchIn(board.A1)
-touch2 = touchio.TouchIn(board.A2)
-touch3 = touchio.TouchIn(board.A3)
-touch4 = touchio.TouchIn(board.A4)
-touch5 = touchio.TouchIn(board.A5)
-touch6 = touchio.TouchIn(board.A6)
-touch7 = touchio.TouchIn(board.A7)
+touch_A1 = touchio.TouchIn(board.A1)
+touch_A2 = touchio.TouchIn(board.A2)
+touch_A3 = touchio.TouchIn(board.A3)
+touch_A4 = touchio.TouchIn(board.A4)
+touch_A5 = touchio.TouchIn(board.A5)
+touch_A6 = touchio.TouchIn(board.A6)
+touch_TX = touchio.TouchIn(board.TX)
```

Comment:
Why TX? Can't we have this as A7 on both boards? Or is it labeled TX?

Reply:
on the bluefruit, it is labeled TX only